prov/verbs/src/ep_rdm/verbs_cq_ep_rdm.c

```diff
@@ -70,6 +70,8 @@ static ssize_t fi_ibv_rdm_tagged_cq_readfrom(struct fid_cq *cq, void *buf,
 		entry[ret].op_context = cq_entry->context;
 		entry[ret].flags = (cq_entry->comp_flags & ~FI_COMPLETION);
 		entry[ret].len = cq_entry->len;
+		entry[ret].buf = (cq_entry->comp_flags & FI_TRANSMIT) ?
+			cq_entry->src_addr : cq_entry->dest_buf;
```

Comment:
Yes, and multi-recv is under development. Also, I do not see the statement that the buffer must be invalid for other cases :) Anyway, we'll have if-statement here. So, I would keep it as is for now.